media/css/firefox/all.less

```diff
@@ -276,6 +276,7 @@ body.nightly {
     }
 
     #language-search {
+        width: @widthTablet - (@gridGutterWidth * 2 * 2.25);
```

Comment:
Hmm, actually - please ignore my suggestion on making the search box wider. It looks like the extra space is reserved for a second (conditional) column. I'm not entirely convinced this column is ever used / shown, but maybe it is best not to disturb it here.

Please just set the width here explicitly to `660px` (this should override the inherited width) :)